recipes/py6s/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "py6s" %}
+{% set version = "1.7.0" %}
+{% set sha256 = "e16cc28e54a59d3908b3902744d261046630e6811cb50b6a13e746f68d421b8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/1c/ea/fdb4fe3e289e3c14e3bdc473f6b6f595ceb476399600fab21c92db97c82a/Py6S-1.7.0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pysolar ==0.6
+    - matplotlib
+    - scipy
+    - numpy
+    - sixs
+  run:
+    - python
+    - pysolar ==0.6
+    - matplotlib
+    - scipy
+    - numpy
+    - sixs
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - Py6S
+    - Py6S.Params
+    - Py6S.SixSHelpers
+
+about:
+  home: http://py6s.rtwilson.com/
+  license: LGPL
+  license_family: LGPL
+
+  summary: 'A wrapper for the 6S Radiative Transfer Model to make it easy to
```

Comment:
This looks better as a `description`. I would use the same short summary from the repo:

```
A Python interface to the 6S Radiative Transfer Model
```